src/Tests/Microsoft.NET.TestFramework/Microsoft.NET.TestFramework.csproj

```diff
@@ -39,12 +39,8 @@
     <_CopyDirectoryBuildTestDependenciesOutput Include="$(ArtifactsTmpDir)Directory.Build.targets" />
   </ItemGroup>
 
-  <Target Name="_CopyDirectoryBuildTestDependencies"
-          AfterTargets="Build"
-          Inputs="@(_CopyDirectoryBuildTestDependenciesInput)"
-          Outputs="@(_CopyDirectoryBuildTestDependenciesOutput)">
-    <Copy SourceFiles="@(_CopyDirectoryBuildTestDependenciesInput)"
-          DestinationFiles="@(_CopyDirectoryBuildTestDependenciesOutput)" />
+  <Target Name="_CopyDirectoryBuildTestDependencies" AfterTargets="Build" Inputs="@(_CopyDirectoryBuildTestDependenciesInput)" Outputs="@(_CopyDirectoryBuildTestDependenciesOutput)">
+    <Copy SourceFiles="@(_CopyDirectoryBuildTestDependenciesInput)" DestinationFiles="@(_CopyDirectoryBuildTestDependenciesOutput)" />
```

Comment:
This looks like it's just a formatting change (probably by Visual Studio).  Revert this.